js/utils/helpers.js

```diff
@@ -2134,8 +2147,10 @@
                     if (attrs && attrs.length == 1 && attrs[0].children) {
                         // Check If the markdown is a link
                         if (attrs[0].children[0].type == "link_open") {
-                            var videoHTML="<video controls ", openingLink = attrs[0].children[0];
-                            var srcHTML="", videoClass='class="' + module._classNames.postLoad, videoAttrs="", flag = true, posTop = true;
+                            var videoHTML="<video controls ", videoClass='class="' + module._classNames.postLoad + " " + "hide-in-print" + " ", openingLink = attrs[0].children[0];
```

Comment:
You don't need the extra space after `hide-in-print`. You should also change the expected values that you changed (and remove the space).